Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Controllers/BrakeController.cs

```diff
@@ -142,8 +144,10 @@ public ScriptedBrakeController(MSTSLocomotive locomotive)
             Locomotive = locomotive;
 
             MaxPressurePSI = 90;
+            MaxOverchargePressurePSI = 95;
```

Comment:
Sane defaults, I presume, or...?

Reply:
A typical overcharge application increases pressure in 5 psi above max pressure. It is intended that content creators use the related .eng parameters to set appropriate values for those variables. For existing stock the brake controller doesn't have an "Overcharge" position, so it will be unused.